miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSequencingParametersService.java

```diff
@@ -148,13 +149,29 @@ private void validateRunType(SequencingParameters params, List<ValidationError>
     }
   }
 
+  private void validateMovieTime(SequencingParameters params, List<ValidationError> errors) {
+    if(params.getInstrumentModel().getPlatformType() == PlatformType.PACBIO) {
+      if(params.getMovieTime() == null) {
+        errors.add(new ValidationError("movieTime", "Movie time value must be specified for "
+            + "PacBio instruments"));
+      }
+      if(params.getMovieTime() < 0) {
```

Comment:
This needs to be `else if`; otherwise, it gets checked even if the movietime is null, leading to the NPE you sent me on Slack.